modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/ContainsFunction.java

```diff
@@ -48,11 +48,7 @@ public Boolean apply(Object object1, Object object2) {
 			return apply(object1.toString(), object2);
 		}
 
-		if ((object1 instanceof String) && (object2 instanceof String)) {
-			return apply((String)object1, (String)object2);
-		}
-
-		return false;
+		return apply(object1.toString(), object2.toString());
```

Comment:
@seltong could you check if `object2` is null (same as `object1` [here](https://github.com/liferay-forms/liferay-portal/blob/31b48ccdc8ec3c4eb5571765b7246f0eebf39555/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/ContainsFunction.java#L37))? Thanks!